lib/index.js

```diff
@@ -7,8 +7,10 @@ const Toys = require('toys');
 const Joi = require('joi');
 const Migrator = require('./migrator');
 const SchwiftyModel = require('./model');
+const SchwiftyMigration = require('schwifty-migration');
 const Schema = require('./schema');
 const Package = require('../package.json');
+const { promisify } = require('util');
```

Comment:
I know it's a little annoying, but we should stick by the hapi style,

```
const Util = require('util');
// ...
Util.promisify(/* ... */);
```

Also good to hoist `require()`s to core node modules up to the top of the file.

Reply:
Ohh right, I think I got used to destructured imports on the front-end